src/modules/api-utils.js

```diff
@@ -0,0 +1,13 @@
+export function getData(url) {
+  return fetch(url).then((res) => res.json());
+}
+
+export function sendData(url, data) {
+  return fetch(url, {
+    method: 'POST',
+    headers: {
```

Comment:
- [ ] Please, consider using ES6 syntax while creating the functions. I have checked all `.js` files, so I did not see arrow functions used. I know, you can easily fix it, so go-go guys! 👍  Please, fix it in all `.js` files 💯

Reply:
done